front/public/locales/fr/operationalStudies/study.json

```diff
@@ -19,6 +19,7 @@
   "errorHappened": "Une erreur est survenue",
   "filesAndLinks": "Documents & liens",
   "filterPlaceholder": "Rechercher…",
+  "infrastructureWithTwoPoints": "Infra : ",
```

Comment:
```suggestion
  "infrastructureWithTwoPoints": "Infra: ",
```

Reply:
This space has been added because there is a difference between French and English